Interni/Verbali Interni/2020_12_18/components/verbale.tex

```diff
@@ -0,0 +1,95 @@
+\section{Informazioni generali}
+
+\begin{itemize}
+
+	\item \textbf{Canale di comunicazione:} Zoom;
+	
+	\item \textbf{Data:} \DataMeeting{};
+	
+	\item \textbf{Ora inizio:} 21;
+	
+	\item \textbf{Ora fine:} 22;
+	
+	\item \textbf{Segretario:} Matteo Pagotto;
+	
+	\item \textbf{Partecipanti:}
+	
+		\begin{itemize}
+		
+			\item Daniele Giachetto;
+			\item Davide Albiero;
+			\item Francesco De Marchi;
+			\item Giosuè Calgaro;
+			\item Lucrezia Gradi;
+			\item Matteo Pagotto;
+			\item Tommaso Poppi.
+				 
+		\end{itemize}
+
+\end{itemize}
+
+\section{Ordine del giorno}
+
+\begin{itemize}
+
+	\item\textbf{Richiesta videoconferenza con l'azienda proponente;}
+	
+	\item\textbf{Discussione punti esposti nella chiamata del 2020-12-17 con il prof. Tullio Vardanega;}
+
+	\item\textbf{Discussione organigramma;}
+
+	\item\textbf{Script Glossario.}
+
+\end{itemize}
+
+\newpage
+
+
+\section{Resoconto}
+
+\subsection{Richiesta videoconferenza con l'azienda proponente}
+
+È stata fatta richiesta via email di un incontro con l'azienda proponente per discutere meglio i requisiti del capitolato \textit{SSD: Soluzioni di sincronizzazione desktop} e la documentazione fornitaci.  
+La mail verrà spedita il giorno 2020-12-21.
+
+\subsection{Discussione chiamata del 2020-12-17}
+
+Sono stati discussi i punti rilevanti emersi durante la chiamata Zoom del 2020-12-17 con il prof. Tullio Vardanega:
+\begin{itemize}
+	\item \textbf{scadenze}: le scadenze diventano date minime di consegna oltre le quali ogni gruppo, previa richiesta, è libero di scegliere le proprie 
+	\item \textbf{documenti}: è stato fissato un numero massimo di pagine per ogni documento e mantenedo i documenti allineati con l'effettivo stato di avanzamento del progetto. 
```

Comment:
La frase risulta leggermente sconnessa, cambiare in qualcosa del tipo [..] per ogni documento ed inoltre si sono stabilite regole per ridurre la zavorra possibile nei documenti, in modo da renderli facilmente migliorabili e più allineati con l'effettivo stato di avanzamento del progetto. " (Ovviamente è un suggerimento scritto di getto, pensaci sopra)